wger/gym/templates/gym/inactive_gym_users_list.html

```diff
@@ -0,0 +1,385 @@
+    </td>
+    <td data-order="{{current_user.last_log|date:'U'}}">
+        {{current_user.last_log|default:'-/-'}}
+    </td>
+    {% if show_gym %}
+    <td>
+        {% if current_user.obj.userprofile.gym_id %}
+            <a href="{{ current_user.obj.userprofile.gym.get_absolute_url }}">
+            {{ current_user.obj.userprofile.gym }}
+            </a>
+        {% else %}
+            -/-
+        {% endif %}
+    </td>
+    {% endif %}
+         <td>
+            <a href="{{ current_user.obj.userprofile.gym.get_absolute_url }}">
+            </a>
+            Inactive
+    </td>
+</tr>
+{% endif %}
+{% endfor %}
+</tbody>
+</table>
+{% endif %}
+
+
+<h4>{% trans "Administrators and trainers" %}</h4>
+<table class="table table-hover">
+<thead>
+<tr>
+    <th style="width: 10%;">{% trans "ID" %}</th>
+    <th style="width: 40%;">{% trans "Username" %}</th>
+    <th>{% trans "Name" %}</th>
+    {% if perms.gym.manage_gym or perms.gym.manage_gyms %}
+        <th style="text-align: right;">{% trans "Roles" %}</th>
+    {% endif %}
+</tr>
+</thead>
+<tbody>
+{% for current_user in object_list.admins %}
+<tr>
+    <td>
+        {{current_user.obj.pk}}
+    </td>
+    <td>
+        {{current_user.obj}}
+
+        {% if current_user.perms.gym_trainer %}
+            <span class="label label-primary">{% trans "Trainer" %}</span>
+        {% endif %}
+
+        {% if current_user.perms.manage_gym %}
+            <span class="label label-primary">{% trans "Gym manager" %}</span>
+        {% endif %}
+
+        {% if current_user.perms.manage_gyms %}
+            <span class="label label-primary">{% trans "General manager" %}</span>
+        {% endif %}
+    </td>
+    <td>
+        {{current_user.obj.get_full_name}}
+    </td>
+
+    {% if perms.gym.manage_gym or perms.gym.manage_gyms %}
+    <td style="text-align: right;">
+        <a href="{% url 'gym:gym:edit-user-permission' current_user.obj.pk %}" class="btn btn-default btn-xs wger-modal-dialog">
+            <span class="{% fa_class 'cog' %}"></span>
+        </a>
+    </td>
+    {% endif %}
+</tr>
+{% empty %}
+<tr>
+    <td colspan="4">{% trans "This gym has no administrators or trainers" %}</td>
+</tr>
+{% endfor %}
+</tbody>
+</table>
+{% endblock %}
+
+
+{% block sidebar %}
+{#             #}
+{# Gym details #}
+{#             #}
+{% if perms.gym.change_gym %}
+<div class="btn-group pull-right">
+    <button type="button" class="btn btn-default dropdown-toggle btn-xs" data-toggle="dropdown">
+        {% trans "Actions" %} <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu" role="menu">
+        <li>
+            <a href="{% url 'gym:gym:edit' gym.id %}" class="wger-modal-dialog">{% trans "Edit"%}</a>
+        </li>
+        <li>
+            <a href="{% url 'gym:export:users' gym.id %}">{% trans "Export"%}</a>
+        </li>
+    </ul>
+</div>
+{% endif %}
+
+<h4>{% trans "Details" %}</h4>
+<table class="table">
+    <tr>
```

Comment:
I used tr and td since they have been used in the existing tables for consistency.